index.js

```diff
@@ -45,10 +45,22 @@ try {
         changes.filter(obj => obj.change.actions[0] === "update").length + ' to change, ' +
         changes.filter(obj => obj.change.actions[0] === "delete").length + ' to destroy.</b>'
 
-    const output = `
+    let output = showFileName ? `\`${inputFile}\`` : ""
+
+    output += `
 <details><summary>${summary}</summary>
 ${message}
-</details>`;
+</details>
+
+`;
+
+    return output;
+}
+
+try {
+    let output = "";
+
+    inputFilenames.forEach(file => output += fileComment(file, inputFilenames.length > 1));
```

Comment:
I think we should use something like [`Array.prototype.reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) to build this instead of using `Array.prototype.forEach` to act as a string builder.

```js
const output = inputFilenames.reduce((str, file) => str + fileComment(file, inputFilenames.length > 1), "");
```